src/did/src/transaction.rs

```diff
@@ -237,50 +236,6 @@ pub struct Transaction {
     pub chain_id: Option<U256>,
 }
 
-impl Transaction {
```

Comment:
why did you remove them? They seem still useful to me

Reply:
I am writing these functions in the evm which takes in required values since we are using the values from execution

Reply:
Shound't you add them to the StorableExecutionResult instead?

Reply:
I thought about that it seems confusing and misrepresentation with what `StorableExecutionResult` does when compared to these methods.

Reply:
But It can be added if you think it's all good!